smarts/sstudio/types.py

```diff
@@ -108,16 +112,31 @@ class LaneChangingModel(_SumoParams):
     """Models how the actor acts with respect to lane changes."""
 
     def __init__(self, **kwargs):
+        warnings.warn(
+            f"The LaneChangingModel has has replaced by {SumoVTypeOverride.__name__}. Instantiating this object type is deprecated.",
+            DeprecationWarning,
+        )
```

Comment:
As you know, I used the `LaneChangingModel` in the `smarts_traffic` branch to allow setting parameters related to cut-ins.  So if you do make the change suggested here, I would rename `SumoVTypeOverride` to just be `VTypeOverride`.  However, I'm not sure even that is general enough because it assumes something called "vtype", which may not always be the case.  (For one thing, note that these apply to traffic _actors_, not just _vehicles_.)